Upstream: submitted, http://www.openldap.org/its/index.cgi?findid=8168

From 5ef01017127746404e592176701018cd20c2b61b Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Thu, 11 Jun 2015 20:44:06 +0200
Subject: [PATCH] Allow passing AR to make

This is helpful when the ar executable is named differently, for
example with an arch specific prefix.
---
 libraries/liblmdb/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/libraries/liblmdb/Makefile b/libraries/liblmdb/Makefile
index 25c1095..759fc79 100644
--- a/libraries/liblmdb/Makefile
+++ b/libraries/liblmdb/Makefile
@@ -17,6 +17,7 @@
 # read mdb.c before changing any of them.
 #
 CC	= gcc
+AR	= ar
 W	= -W -Wall -Wno-unused-parameter -Wbad-function-cast -Wuninitialized
 THREADS = -pthread
 OPT = -O2 -g
@@ -48,7 +49,7 @@ test:	all
 	./mtest && ./mdb_stat testdb
 
 liblmdb.a:	mdb.o midl.o
-	ar rs $@ mdb.o midl.o
+	$(AR) rs $@ mdb.o midl.o
 
 liblmdb.so:	mdb.o midl.o
 #	$(CC) $(LDFLAGS) -pthread -shared -Wl,-Bsymbolic -o $@ mdb.o midl.o $(SOLIBS)
-- 
2.4.3

